app/javascript/packs/selectors/answersSelector.js

```diff
@@ -1,3 +1,22 @@
 export function getAnswerForQuestionAndBuilding(questionId, buildingId, state) {
-  return state.buildings[buildingId].answers[questionId]
+    return state.buildings[buildingId].answers[questionId]
 }
+
+//returns (from the questions provided) how many are remaining to answer
+export function getRemainingAnswersforCategory(questions, buildingId, state) {
+    if (buildingId) {
```

Comment:
To make this a little less nested, can we specify the failing conditions first? i.e.

```javascript
if (!buildingId) return null;
if (!questions) return 0;
< the rest here, no nesting :) >
```